### DISCUSSION TITLE
Multiports udp and tcp

### DISCUSSION THREAD

**Discussion by josep112** [nginx-proxy/nginx-proxy#1625] [Q&A]
First of all, I'm sorry to write here on a bug but the pagehttps: //groups.google.com/forum/#! Forum / nginx-proxy doesn't seem to work anymore

My doubt is if it is possible to redirect multiple udp ports through VIRTUAL_PORT, in this case it would be 5060, 10000-1099 udp ports,  443, 80 tcp

**Comment by tkw1536** [comment]
I'm not sure what you're trying to achieve. 

Do you want to forward non-http UDP and TCP ports into containers based on hostname? Hostnames only exist in http(s) and as such this can't be done generically. 

If you want to redirect specific host udp ports into a specific container you can (independently of nginx-proxy) just use normal docker networking features.

**Comment by josep112** [comment]
hi,
The idea is to direct everything that comes in DNS for the container, just as a single port 80.443 can be directed to several containers would like to use UDP ports too
